master/getting-started/kubernetes/installation/vagrant/Vagrantfile

```diff
@@ -1,3 +1,7 @@
+---
+layout: null
```

Comment:
It wasn't already there. https://github.com/projectcalico/calico/blob/master/master/getting-started/kubernetes/installation/vagrant/Vagrantfile